gbpservice/nfp/orchestrator/drivers/orchestration_driver.py

```diff
@@ -634,6 +640,10 @@ def create_network_function_device(self, device_data,
                                           by=len(interfaces))
             return None
 
+        vendor_data = vendor_data_result.get('result', None)
+        if not vendor_data:
+            LOG.exception(_LE('Failed to get vendor data for device creation.'))
+
```

Comment:
should this be exception ? I think we discussed this last time, vyos will not have vendor data right.. I think it is ok not to have vendor data, so debug/warn should be fine i suppose ?